src/components/buttons/primary-button/primary-button.js

```diff
@@ -1,48 +1,51 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import classnames from 'classnames';
 import isNil from 'lodash.isnil';
+import { css } from '@emotion/core';
+import vars from '../../../../materials/custom-properties';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
+import Spacings from '../../spacings';
 import AccessibleButton from '../accessible-button';
-import styles from './primary-button.mod.css';
-
-const getButtonClassNames = ({ isToggled, isDisabled }) => {
-  if (isDisabled) return styles.disabled;
-  if (isToggled) return styles.active;
-  return styles.button;
-};
+import {
+  getButtonLayoutStyles,
+  getButtonStyles,
+} from './primary-button.styles';
 
 const PrimaryButton = props => {
   const dataProps = {
     'data-track-component': 'PrimaryButton',
     ...filterDataAttributes(props),
   };
+  const isActive = props.isToggleButton && props.isToggled;
   return (
-    <div
-      className={classnames(styles[`button-layout-${props.size}`], {
-        [styles[`button-tone-${props.tone}`]]: !props.isDisabled,
-      })}
-    >
+    <div css={getButtonLayoutStyles(props.size)}>
       <AccessibleButton
         type={props.type}
         buttonAttributes={dataProps}
         label={props.label}
         onClick={props.onClick}
+        isToggleButton={props.isToggleButton}
         isToggled={props.isToggled}
         isDisabled={props.isDisabled}
-        className={getButtonClassNames({
-          isToggled: props.isToggled,
-          isDisabled: props.isDisabled,
-        })}
+        css={getButtonStyles(props.isDisabled, isActive, props.tone)}
       >
-        {Boolean(props.iconLeft) && (
-          <span className={styles['icon-container']}>
-            {React.cloneElement(props.iconLeft, {
-              theme: props.isDisabled ? 'grey' : 'white',
-            })}
-          </span>
-        )}
-        <span>{props.label}</span>
+        <Spacings.Inline alignItems="center" scale="xs">
```

Comment:
I saw that this was used in the SecondaryButton and figured it should be used here as well

Reply:
I guess this relates to this VRT diff

![image](https://user-images.githubusercontent.com/1110551/51553262-abdcf380-1e72-11e9-8d5e-fdebb47c04af.png)

I think it should be fine to accept this. /cc @montezume

Reply:
Same for this one:

![image](https://user-images.githubusercontent.com/1110551/51553325-d6c74780-1e72-11e9-91b9-55df54da6a91.png)